draft-thomson-httpbis-h2-stream-limits.md

```diff
@@ -204,6 +204,13 @@ RFC9113}}).  When a `MAX_STREAMS` frame is received, the endpoint MUST
 subsequently only create streams according to the rules in {{limits}} and
 ignore any value of the `SETTINGS_MAX_CONCURRENT_STREAMS` setting.
 
+For compatibility with existing implementations which might not support
+`MAX_STREAMS`, endpoints can set `SETTINGS_MAX_CONCURRENT_STREAMS` to a value
+less than or equal to the number of concurrent streams intended to be allowed
+with `MAX_STREAMS`.  An endpoint MUST NOT enforce a `MAX_STREAMS` limit lower
```

Comment:
I an not sure about this "MUST NOT".  It might be better to just explain the consequences of failing to do this correctly.